Code/GraphMol/FileParsers/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ rdkit_library(FileParsers
               TDTMolSupplier.cpp TDTWriter.cpp
               TplFileParser.cpp TplFileWriter.cpp
               PDBParser.cpp PDBWriter.cpp PDBSupplier.cpp
+              MaeMolSupplier.cpp
```

Comment:
Since the maeparser isn’t always included, we should make this optional.

Reply:
Done, and chased down other places I needed to hide it too